src/pages/guides/changelog/index.md

```diff
@@ -22,6 +22,13 @@ contributors:
 
 # Changelog
 
+## [2.0.1] - 2022-12-20
+
+### Added
+
+- `initialize()` method is now async
```

Comment:
May be updated basis https://cclight.slack.com/archives/C02NNKC76K0/p1675405522163229 info.